### PR TITLE
Change contact form recipients to lowerCamelCase to match with store

### PR DESCRIPTION
**Pull request by danieljames-dj**
This is a sub-PR of https://github.com/thewca/worldcubeassociation.org/pull/9282, expecting that PR will take some time for the review to finish. Also, merging this PR will make that PR simpler.

### CODE REVIEW

File: config/locales/en.yml
Comment (gregorbg) [inline config/locales/en.yml:1]: Not a fan of camelCase in the locale files. YML should use snake_case by convention.

**Review by gregorbg — COMMENTED** [review]
I understand your intention behind this PR, but perhaps it would be best to stick with snake_case in the backend and consider something like lodash case converters for the frontend?

https://lodash.com/docs/4.17.15#snakeCase
https://lodash.com/docs/4.17.15#camelCase

File: config/locales/en.yml
Comment (danieljames-dj) [reply config/locales/en.yml:1]: Changed

**Comment by danieljames-dj** [comment]
I didn't knew about lodash helper method, that helped. Made the changes, thank you.

File: app/webpacker/components/ContactsPage/ContactForm.jsx [hunk 1]
```diff
@@ -16,12 +16,15 @@ import Competition from './SubForms/Competition';
...
+  'communicationsTeam',
+  'resultsTeam',
+  'softwareTeam',
 ];
 
-const CONTACT_RECIPIENTS_MAP = _.keyBy(CONTACT_RECIPIENTS);
+const CONTACT_RECIPIENTS_MAP = CONTACT_RECIPIENTS.reduce((acc, recipient) => {
+  acc[recipient] = _.snakeCase(recipient);
+  return acc;
+}, {});
```
Comment (gregorbg) [inline app/webpacker/components/ContactsPage/ContactForm.jsx:27]:
```suggestion
const CONTACT_RECIPIENTS_MAP = _.keyBy(CONTACT_RECIPIENTS, _.camelCase);
```

Comment (gregorbg) [reply app/webpacker/components/ContactsPage/ContactForm.jsx:27]: **NOTE** This only works if you undo the diff above and keep `CONTACT_RECIPIENTS` in snake_case.
Lodash only has a method for turning array into object when you create keys of the object and retain the values. There is no equivalent shorthand method for iterating over an arrays as keys and determining values.

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: app/webpacker/components/ContactsPage/ContactForm.jsx (hunk 1 above)
Comment (danieljames-dj) [reply app/webpacker/components/ContactsPage/ContactForm.jsx]: Done, thanks

**Review by gregorbg — APPROVED** [review]

**Pull request merged**